lambda_function/src/file_processor/file_processor.py

```diff
@@ -207,6 +241,9 @@ def _process_file(self) -> None:
                 science_file = science_filename_parser(parsed_file_key)
                 this_instr = science_file["instrument"]
                 destination_bucket = get_instrument_bucket(this_instr, self.environment)
+                if self.tracker and science_file["level"] == "l0":
+                    # If level is L0 should be tracked in CDF
```

Comment:
Why is the tracking for L0 files different than others?

Reply:
This is to catch the `l0` file added event and add it to the table. I think keeping the CDFTracker in this one function is cleaner.

The flow is like this:
`l0` -> `l1` - Should add both the `l0` and `l1` files to the tables
`l1` -> `ql` - Should add `ql` 
`ql` -> `l{n}` - Should add whatever the next level is after quicklook